.github/workflows/development.yml

```diff
@@ -0,0 +1,37 @@
+name: (CirrusCI) Development
+
+on:  workflow_dispatch
```

Comment:
but this workflow is also manual

Reply:
both, manual and on push, I still want the option to trigger it manually

Reply:
though I don't see a way to test it before merging (github action)

Reply:
but if you specify 'workflow_dispatch' then if wouldn't trigger on push to develop IMO